src/lib/components/Modal/InviteToWorkspace.svelte

```diff
@@ -0,0 +1,296 @@
+          color: "whiteColor",
+        },
+        {
+          name: "Viewer",
+          id: WorkspaceRole.WORKSPACE_VIEWER,
+          description: "View Resources within a workspace.",
+          color: "whiteColor",
+        },
+      ]}
+      method={selectedRole ? selectedRole : ""}
+      onclick={handleDropdown}
+    />
+    {#if showErrors && selectedRole === "select"}
+    <p class="error-text">Role Cannot Be Empty</p>
+  {/if}
+  </div>
+  <div class="text-textColor mt-2" style="font-size: 12px;">
+    You can invite your team members or external collaborators to a this
+    workspace. Invited people will have access to only the <span
+      style="color:white">{currentWorkspaceDetails.name}</span
+    > workspace.
+  </div>
+  <div class="d-flex align-items-center justify-content-between">
+    <div class="description mt-4">
+      <p class="mb-0 text-textColor">
+        Workspace<span style="color:white"
+          >: {currentWorkspaceDetails.name}</span
+        >
+      </p>
+      <p class="text-textColor">
+        Team:<span style="color:white">: {teamName}</span>
+      </p>
+    </div>
+    <div>
+      <button
+        class="invite-btn btn rounded border-0 py-2"
+        on:click={handleInvite}>Send Invite</button
+      >
+    </div>
+  </div>
+</div>
+
+<style>
+  .background-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background: var(--background-hover);
+    backdrop-filter: blur(3px);
+    z-index: 12;
+  }
+  .container {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    height: 50%;
+    width: 60%;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: var(--background-color);
+    z-index: 12;
+    padding: 2%;
+    border-radius: 10px;
+  }
+  .header-title {
+    font-size: 20px;
+  }
+  .invite-header {
+    font-size: 14px;
+  }
+  .invite-subheader {
+    font-size: 12px;
+  }
+  .description {
+    font-size: 12px;
+  }
+  .asterik {
+    color: var(--dangerColor);
+    margin-left: 4px;
+  }
+  .btn-close1 {
+    background-color: var(--background-color);
+  }
+  .btn-close1:hover {
+    background-color: var(--background-dropdown);
+  }
+  .btn-close1:active {
+    background-color: var(--dangerColor);
+  }
+  .invite-btn {
+    background-color: var(--primary-btn-color);
+  }
+  .invite-btn:hover {
+    background: var(--send1-hoverbutton);
+    color: var(--white-color);
+  }
+  .error-text {
+    margin-top: 4px;
+    color: var(--error--color);
+    font-size: 12px;
+  }
+  .email-container {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: transparent;
+    border: 1px solid var(--border-color);
+    padding: 3px 5px 3px 5px;
+  }
+  #input-email {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    gap: 5px;
+  }
+  @media (min-width: 1000px) {
+    .container {
+      display: flex;
```

Comment:
try to add classes directly to the div where this class is used and add bootstrap classes directly